src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -43,6 +43,12 @@ public static class MSBuildLocator
         /// </summary>
         public static bool IsRegistered => s_registeredHandler != null;
 
+        /// <summary>
+        ///     This flag enables the removal of the existing restriction on querying installed Visual Studio instances 
+        ///     with a runtime version lower than or equal to the one in the current environment.
+        /// </summary>
```

Comment:
I would like to rephrase this to be scarier :)

```suggestion
        /// <summary>
        ///     Allow discovery of .NET SDK versions that are unlikely to be successfully loaded in the current process.
        /// </summary>
        /// <remarks>
        ///     Defaults to <see langword="false"/>. Set this to <see langword="true"/> only if your application has special logic to handle loading an incompatible SDK, such as launching a new process with the target SDK's runtime.
        /// </remarks.
```